chainservice/chainservice.go

```diff
@@ -224,6 +224,7 @@ func createBlockSyncer(
 			}
 			if len(peers) == 0 {
 				log.L().Error("no peers")
+				return
```

Comment:
can delete this func entirely, it is not used

Reply:
this could be done in another pr